src/app/bitstream-page/bitstream-authorizations/bitstream-authorizations.component.ts

```diff
@@ -19,7 +19,7 @@ import { DSpaceObject } from '../../core/shared/dspace-object.model';
 import { ResourcePoliciesComponent } from '../../shared/resource-policies/resource-policies.component';
 
 @Component({
-  selector: 'ds-collection-authorizations',
+  selector: 'ds-bitstream-authorizations',
```

Comment:
This is very odd that this component has been using the wrong `selector` for so long (it seems to also be like this in 7.x and 8.x).  Especially since we have a separate component that is **also using** `ds-collection-authorizations : https://github.com/DSpace/dspace-angular/blob/main/src/app/collection-page/edit-collection-page/collection-authorizations/collection-authorizations.component.ts#L18

This is definitely a bug.  I was surprised it's not been found yet, but it seems that both of these components are only referenced via routes (e.g. this BitstreamAuthorizationComponent is loaded via [`bitstream-page-routes.ts`](https://github.com/DSpace/dspace-angular/blob/main/src/app/bitstream-page/bitstream-page-routes.ts#L80)).  So, that's why we didn't notice this invalid selector earlier...the selector isn't being used right now.